templates/sag-db-sqlserver/README.md

```diff
@@ -18,8 +18,7 @@ With this template you can create user, database, and webMethods database schema
 
 ## Running as a standalone Composite Template
 
-Run this command to install Database Component Configurator on `local` 10.2 managed node and create
-webMethods `webm` database, `webm` user and all schemas for 10.1 release on `sqlserver` SQL server connecting with `sa` user and `MaNaGe123` password:
+To install Database Component Configurator on a node with alias `local` and version 10.2 and create a webMethods database, named `webm` with credentials `webm`/`webm` and all database schemas for version 10.2 on a Microsoft SQL Server host connecting with `sa` /`MaNaGe123`: 
 
 ```bash
 sagcc exec templates composite apply sag-db-sqlserver
```

Comment:
sagcc exec templates composite apply sag-db-sqlserver \
  release=10.1 repo.product=products-10.1 repo.fix=fixes-10.1 nodes=local \
  db.host=sqlserver db.admin.username=sa db.admin.password=MaNaGe123 \
  db.name=webm db.username=webm db.password=webm \
  db.products=[IS,MWS] \
   --sync-job --wait 360